src/NodeApi.DotNetHost/TypeExporter.cs

```diff
@@ -212,7 +212,14 @@ private void RegisterDerivedType(TypeProxy derivedType, Type? baseOrInterfaceTyp
         {
             string baseOrInterfaceTypeName = TypeProxy.GetTypeProxyName(baseOrInterfaceType);
 
+            if (baseOrInterfaceType.Namespace == null)
+            {
+                Trace($"Skipping un-namespaced base or interface: {baseOrInterfaceType.Name}");
+                return;
+            }
+
             NamespaceProxy? ns = GetNamespaceProxy(baseOrInterfaceType.Namespace!);
```

Comment:
Did you consider updating `GetNamespaceProxy()` to handle a null namespace value instead? It could simply use an empty string instead when the value is null. I think that might be better, considering it will need to support (not skip) un-namespaced types soon anyway for #262.

Reply:
Ready